app/Http/Controllers/ServerController.php

```diff
@@ -38,6 +33,16 @@ public function create()
             return redirect()->route('servers.index')->with('error', "You've already reached your server limit!");
         }
 
+        //Required Verification for creating an server
+        if(Configuration::getValueByKey('VERIFIED_EMAIL_TO_MAKE_SERVER') === 'true' && !Auth::user()->hasVerifiedEmail()) {
+            return redirect()->route('profile.index')->with('error', "You havent verified your email! Thats required to create an server.");
+        }
+
+        //Required Verification for creating an server
+        if(Configuration::getValueByKey('VERIFIED_DISCORD_TO_MAKE_SERVER') === 'true' && !Auth::user()->discordUser) {
```

Comment:
check comment above :)
To what method have you added this script? should be added to both the create and store method to prevent any bypasses

Reply:
Done in new PR (#95) (sadly I was not able to commit into this one)